specification/azsadmin/resource-manager/compute/Microsoft.Compute.Admin/preview/2021-01-01/Quotas.json

```diff
@@ -0,0 +1,315 @@
+          }
+        },
+        "tags": [
+          "Quotas"
+        ],
+        "description": "Creates or Updates a Compute Quota with the provided quota parameters.",
+        "summary": "Creates or Updates a Compute Quota.",
+        "operationId": "Quotas_CreateOrUpdate",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../2015-12-01-preview/Compute.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../2015-12-01-preview/Compute.json#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "Quotas.json#/parameters/QuotaParameter"
+          },
+          {
+            "$ref": "Quotas.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "Quotas.json#/parameters/NewQuotaParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "Quotas.json#/definitions/Quota"
+            }
+          }
+        }
+      },
+      "delete": {
+        "x-ms-examples": {
+          "Deletes specified Compute quota.": {
+            "$ref": "examples/Quotas/Delete.json"
+          }
+        },
+        "tags": [
+          "Quotas"
+        ],
+        "summary": "Deletes specified Compute quota",
+        "description": "Delete an existing Compute quota.",
+        "operationId": "Quotas_Delete",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../2015-12-01-preview/Compute.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../2015-12-01-preview/Compute.json#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "Quotas.json#/parameters/QuotaParameter"
+          },
+          {
+            "$ref": "Quotas.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Compute.Admin/locations/{location}/quotas": {
+      "get": {
+        "x-ms-examples": {
+          "Get a list of existing Compute quotas.": {
+            "$ref": "examples/Quotas/List.json"
+          }
+        },
+        "tags": [
+          "Quotas"
+        ],
+        "summary": "Lists all Compute quotas.",
+        "description": "Get a list of existing Compute quotas.",
+        "operationId": "Quotas_List",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../2015-12-01-preview/Compute.json#/parameters/SubscriptionIdParameter"
```

Comment:
please make a copy of compute.json in stable version and refer to that here. Thanks!